CommonTools/BaseParticlePropagator/src/BaseParticlePropagator.cc

```diff
@@ -506,11 +506,10 @@ bool BaseParticlePropagator::propagateToHcalEntrance(bool first) {
 
 bool BaseParticlePropagator::propagateToVFcalEntrance(bool first) {
   //
-  // Propagation to VFCAL entrance
-  // TODO: include proper geometry
-  // Geometry taken from DAQ TDR Chapter 13
+  // Propagation to VFCAL (HF) entrance
+  // Geometry taken from xml: Geometry/CMSCommonData/data/cms.xml
 
-  setPropagationConditions(400.0, 1110.0, first);
+  setPropagationConditions(400.0, 1114.95, first);
```

Comment:
as a future development item, it would be even better to get these directly from the geometry instead of hardcoding the values

Reply:
although this is a number that is not supposed to change, I agree with @kpedro88 that this implementation is intrinsicattly weak, although its fix can be deferred to a second moment

Reply:
I understand. If we make this update, I want to update a few other hardcoded numbers, but some hardcoded numbers may not be directly extractable from geometry. It's not a trivial change, so I prefer to keep this separate, while fixing this known small inconsistency for now.